mod.conf

```diff
@@ -1,4 +1,4 @@
 name = moretrees
-depends = default, biome_lib, vessels
```

Comment:
ethereal should be an optional depend to keep trees registered (https://notabug.org/TenPlus1/ethereal/src/master/biomes_init.lua#L26)

Reply:
looks like i need to have a word with tenplus1 again....

Reply:
https://github.com/mt-mods/moretrees/pull/11/commits/8570aab0c91ca74ba8d443b4b345b0ea21475d69 resolved